assets/python/ml/pyml:setup_variables_packages.j2.py

```diff
@@ -32,7 +32,7 @@
 target_column_name = "target"
 
 # The type of ML problem being performed. Can be either "Regression", "Classification," or "Clustering."
```

Comment:
stated allowed values don't match the case of the actual value

Reply:
Also, why is this value hard-coded like this?  What's the behavior here?  I guess this file is the regression test asset? Then the filename should be changed to be similar to the clustering and classiffication files

Reply:
The regression test asset is in its own file for now - the test changes the value by itself. Initially, the intent here was to have one settings.py file that users could edit.

When translating this into the regression/clustering/classificaiton default workflows, there wasn't a good way to just change the value on-the-fly. I've made a task to do this more programatically here. We're going to be making it into an 'important setting' that just gets templated in: https://exabyte.atlassian.net/browse/SOF-4401

Reply:
So why isn't this file named `assets/python/ml/pyml:setup_regression_variables_packages.j2.py` ?